windows.md

```diff
@@ -390,6 +390,41 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    When opening Docker Desktop during Option A or Option B, an error message `Docker Desktop - Windows Hypervisor is not present` may appear:<br><br>
+
+    <img src="windows-7-docker-desktop-error.avif"><br><br>
+
+    If this appears for you, follow the next steps to enable virtualization on your machine (if you don't receive the error, you can skip to the Docker testing step):
+
+    Check if your machine supports virtualization by running the following command on PowerShell as an administrator:
+
+    ```bash
+    Get-ComputerInfo -Property HyperV*
+    ```
+
+    This should display the following output:
+
+    ```bash
+    HyperVisorPresent: False
+    HyperVRequirementDataExecutionPreventionAvailable : True
+    HyperVRequirementSecondLevelAddressTranslation    : True
+    HyperVRequirementVirtualizationFirmwareEnabled    : False
+    HyperVRequirementVMMonitorModeExtensions          : True
+    ```
+
+    `HyperVRequirementVMMonitorModeExtensions: True` means that your machine supports virtualization and `HyperVRequirementVirtualizationFirmwareEnabled: False` means that virtualization is not enabled
+
+    If your output is different than the above, please contact the UpLeveled team on Slack for further assistance
+
+    Enable virtualization in your BIOS / UEFI settings by following these steps:
```

Comment:
once you [make the "Check if your machine supports virtualization" the new step 1 above](https://github.com/upleveled/system-setup/pull/16/files/ca0070e6b6f4bd4d06ccdd96c6f0f32cff8a15da#r1561132600), then you can probably remove this paragraph